modules/documents/app/controllers/documents_controller.rb

```diff
@@ -155,7 +155,8 @@ def update_header_component_via_turbo_stream(state: :show)
   end
 
   def respond_with_document_update_turbo_streams(service_call)
-    update_via_turbo_stream(component: Documents::ContentEditableComponent.new(@document, project: @project))
+    @document = service_call.result
+    update_header_component_via_turbo_stream(state: :show)
```

Comment:
I switched to updating only the header because saving would clear the description when collaboration was enabled.